src/Tooltip.jsx

```diff
@@ -1,35 +1,32 @@
 /**
  * @class Tooltip
- * @description A lightweight and responsive tooltip.
+ * @description The container of a lightweight and responsive tooltip.
  */
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Portal from 'react-minimalist-portal';
-import positions from './position';
-
-// default colors
-const defaultColor = '#fff';
-const defaultBg = '#333';
-
-const stopProp = e => e.stopPropagation();
+import TooltipBubble from './TooltipBubble';
 
 class Tooltip extends React.PureComponent {
   static propTypes = {
+    // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
What is unused? Why? Can we drop this comment?

Reply:
I'll try to look - I'd want to document the properties somewhere anyway, where would you suggest? Inline here with comments or revise/update `README.md`? Also, if possible, after we're done, I'd update the `index.d.ts` as well